tests_e2e/test_suites/images.yml

```diff
@@ -55,6 +55,19 @@ image-sets:
       - "oracle_610"
       - "rhel_610"
 
+   # These are the distros which have periodic log collector support.
+   log-collector-endorsed:
+      - "centos_82"
+      - "rhel_82"
+      - "ubuntu_1604"
+      - "ubuntu_1804"
+      - "ubuntu_2004"
+      - "ubuntu_2204"
+      - "ubuntu_2204_minimal"
+      - "ubuntu_2204_arm64"
+      - "ubuntu_2404"
+      - "ubuntu_2404_minimal"
```

Comment:
Note: when v2 enabled for log collector, we should add other distros where v2 is default